Assignment2/Saabertooth1-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,62 @@
+{
+	"@context": {
+			"el": "http://example.org/elements",
+			"xsd": "http://www.w3.org/2001/XMLSchema",
+			"class": {
+              "@id": "http://example.org/class",
+              "@type": "el:class"
+			},
+			"sensor": {
+              "@id": "http://example.org/sensor",
+			  "@type": "el:sensor"
+			},
+			"computer": {
+              "@id": "http://example.org/computer",
+              "@type": "el:computer"
+			},
+			"measurement": {
+              "@id": "http://example.org/mesurement",
+              "@type": "el:measurement"
+			},
+			"user": {
+              "@id": "http://example.org/user",
+              "@type": "el:user"
+			},
+      		"temperature": {
+              "@id": "http://example.org/temperature",
+              "@type" : "xsd:integer"
+            },
+      		"time": {
+              "@id": "http://example.org/time",
+              "@type": "xsd:dateTime"
+            },
+      		"name": {
+              "@id": "http://example.org/name",
+              "@type": "xsd:string"
+    		}
+      		
+   	},
+  "Class01": {
+    "@id": "http://example.org/Class01",
+    "@type": "class",
+    "Sensor029": {
```

Comment:
It is missing the attribute "includes" that represents the relationship (predicate) between your Class01 (suject) and the other objects (Sensor029 and Computer101), thus you should define it properly. Is something similar to the "geometry" property of the WiFi point example. In addition, you might define objects (Sensor029 and Computer101) into an array, e.g. "includes":[{Sensor029 attributes},{Computer101 attributes}]. Also, the other relatioships are missing (hasOwner, hasMeasurement)